portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SyncSitesAndChannels.testcase

```diff
@@ -245,6 +245,160 @@ definition {
 		}
 	}
 
+	@description = "Bug: LRAC-12938 | Automation ID: LRAC-13010 | Test Summary: Check that event not being sent when desynchronized sites"
+	@priority = 4
+	test CheckEventNotBeingSentWhenDesynchronizedSites {
+		property proxy.server.enabled = "true";
+
+		task ("Connect the DXP to AC") {
+			ACDXPSettings.connectAnalyticsCloud();
+
+			ACDXPSettings.getAssignedPropertyName();
+		}
+
+		task ("Sync channels and sites") {
+			ACDXPSettings.syncChannel(
+				channelName = "Minium Demo Portal",
+				propertyName = ${assignedPropertyName});
+
+			for (var siteName : list "Site Name,Minium Demo") {
+				ACDXPSettings.syncSites(
+					propertyName = ${assignedPropertyName},
+					siteName = ${siteName});
+			}
+		}
+
+		task ("Sync contacts and accounts, sync the attributes, save and finish") {
+			Click(
+				key_text = "Next",
+				locator1 = "Button#ANY");
+
+			ACDXPSettings.syncContactsAndAccountstData();
+
+			Click(
+				key_text = "Next",
+				locator1 = "Button#ANY");
+
+			Click(
+				key_text = "Finish",
+				locator1 = "Button#ANY");
+		}
```

Comment:
Hey @manuferreira, I think you could simplify these lines of code using `ACDXPSettings.connectDXPtoAnalyticsCloud` macro. I think if you are going to synchronize two sites, so you don't need to pass the name of any site to the macro, that way it will select the checkbox of all items visible on the screen and will select your two sites. What do you think?

Reply:
Sounds good!